### PR TITLE
Test changed readme

### PR DESCRIPTION
**Pull request by gforghieri**

### CODE REVIEW

**Review by VictorPrins — COMMENTED** [review]
Can you finish the sentence with a . please

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Readme text file
+
```
Comment (VictorPrins) [inline README.md:2]: Can you remove the blank line here?

**Review by VictorPrins — CHANGES_REQUESTED** [review]
See my comments

File: README.md (hunk 1 above)
Comment (gforghieri) [reply README.md:2]: What's your problem?

**Review by gforghieri — COMMENTED** [review]
See my comments.

**Review by VictorPrins — APPROVED** [review]

**Pull request merged**